Cindy/org detail page

cindyxzhang opened the pull request:
_Describe your feature here_
- shows the individual page for an organization

currently running into the following problems:
- getting the cover image to show up: FIXED
- spacing for project cards :FIXED
- edit button styling (tried material UI but shows up as purple): FIXED
## Related PRs

_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_

## Migrations

_Optional - if you added anything to the database through migration(s)_

## Screenshots

_Required for visual changes_

CC: @kwfk

cindyxzhang committed:
feat(api works): api works

cindyxzhang committed:
Merge branch 'main' into cindy/org-detail-page

cindyxzhang committed:
feat(organizations page): organizations page for individual org

cindyxzhang committed:
feat(edited new format for elizabeth): new format from elizabeths designs

cindyxzhang committed:
feat(finished button styling): finished button styling

cindyxzhang committed:
fix(removed unecessary imports): removed unecessary imports

kwfk commented:
remember to add screenshots when it's a visual task

kwfk commented on pages/orgs/[id].tsx:
use `getServerSideProps` instead

kwfk commented on pages/orgs/[id].tsx:
change item to something more intuitive like org

kwfk commented on pages/orgs/[id].tsx:
change Component name to match function

kwfk commented on pages/orgs/[id].tsx:
change component name to match function

kwfk commented on pages/orgs/[id].tsx:
why is "Next.js + Typescript Example always appended lol

kwfk commented on pages/orgs/[id].tsx:
catch no org error by redirecting to 404 page

kwfk commented on pages/orgs/[id].tsx:
you can replace this with the data you're getting from already
just remember to take care of null cases

kwfk commented on pages/orgs/[id].tsx:
if you're using await, you don't need to use chaining
use one or the other

cindyxzhang committed:
Merge branch 'main' of https://github.com/calblueprint/nbjc into cindy/org-detail-page

cindyxzhang committed:
feat(trying to merge main): merging main

cindyxzhang committed:
feat(fixed server props): fixed server props

kwfk commented on pages/orgs/[id].tsx:
```diff
@@ -0,0 +1,237 @@
+import { useState } from 'react';
+import { GetStaticProps, GetStaticPaths, GetServerSideProps } from 'next';
```
remove unused imports

kwfk commented on pages/orgs/[id].tsx:
```diff
@@ -0,0 +1,237 @@
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { Organization } from '@prisma/client';
+import { sampleOrgData } from 'utils/sample-data';
```
remove unused imports

kwfk committed:
Merge branch 'main' into cindy/org-detail-page

kwfk committed:
feat(org profile): fill fields with backend data

Populated fields with data from the database. Updated styling as well.

kwfk committed:
Merge branch 'main' into cindy/org-detail-page

kwfk committed:
Merge branch 'main' into cindy/org-detail-page

kwfk committed:
feat(homepage and org profile): connect homepage and org profile

Add linking from homepage to the proper organization profile. Also, clean up data in org profile

kwfk committed:
fix(org profile): update spacing styling
